openide.dialogs/src/org/openide/WizardDescriptor.java

```diff
@@ -813,7 +813,7 @@ private static void checkComponent(Panel<?> pnl) {
     }
 
     private <A> void updateStateOpen(SettingsAndIterator<A> data) {
-        if( !initialized.get() ) //#220286
+        if( !initialized.get()) //#220286
```

Comment:
Correcting half of Petr Hřebejk's syntax. Why just half?

Reply:
Ouch; didn't intend to correct at all (leftover from prototyping), but I will fix the rest.